src/test/java/jewellery/inventory/integration/ResourceInUserCrudIntegrationTest.java

```diff
@@ -133,7 +142,9 @@ void updateResourceInUserSuccessfullyWhenAddedMultipleTimes() {
         response.getBody().getResourcesAndQuantities();
     assertNotNull(resourceQuantities);
     assertEquals(1, resourceQuantities.size());
-    assertEquals(RESOURCE_QUANTITY * 2, resourceQuantities.get(0).getQuantity(), 0.001);
+    assertEquals(
+        RESOURCE_QUANTITY.multiply(new BigDecimal("2")).setScale(2, RoundingMode.HALF_UP),
```

Comment:
Another new BigDecimal. Should we try to get rid of all of these and just create them from the util class?

Reply:
I found them all and fixed all but one. In the object mapper config in the SystemEventTestHelper for some reason if I change it to use the utility method there is a test that starts failing...